cpp/src/arrow/compute/kernels/scalar_string_test.cc

```diff
@@ -2640,4 +2640,10 @@ TEST(TestStringKernels, UnicodeLibraryAssumptions) {
 }
 #endif
 
+TEST(TestUtf8IsDigit, SupportsUnicodeDigits) {
```

Comment:
```suggestion
TEST(TestStringKernels, Utf8IsDigit) {
```

BTW, should we update existing (but empty) one instead of adding a new test?
https://github.com/apache/arrow/blob/19439118ce25620b9773fe3616aa1e4dab4fd098/cpp/src/arrow/compute/kernels/scalar_string_test.cc#L1386-L1391

Reply:
> BTW, should we update existing (but empty) one instead of adding a new test?
Thanks for pointing that out, I updated the test in the existing block